### DISCUSSION TITLE
Alignment of figure captions

### DISCUSSION THREAD

**Discussion by suryadheeshjith** [mmistakes/minimal-mistakes#2470] [Q&A] [answered]
I have tried to update the CSS in the file _base.scss by augmenting a text-align: center to the figcaptions but it doesn't seem to work for me. Am I missing something?

```
figcaption {
  margin-bottom: 0.5em;
  color: $muted-text-color;
  font-family: $caption-font-family;
  font-size: $type-size-6;
  text-align : center;
```

**Comment by mmistakes** [comment]
Do you have a link to an example on your site where this is not working? `text-align: center` is the correct CSS. Perhaps you haven't customized the theme's CSS files properly and it's not loading your changes. Without seeing an example I can't really see either way.

**Comment by suryadheeshjith** [comment]
*Author has hidden this message*

**Comment by iBug** [comment, marked as answer]
The `<figcaption>` has `display: block` so what you'd want to apply is:

```css
margin-left: auto;
margin-right: auto;
```

**Comment by suryadheeshjith** [comment]
Yepp that did it! Thank you.
